plugins/c8y_log_plugin/src/main.rs

```diff
@@ -225,17 +225,18 @@ fn create_init_logs_directories_and_files(
 
     // creating c8y-log-plugin.toml
     let logs_path = format!("{logs_dir}/tedge/agent/software-*");
-    let data = toml::toml! {
-        files = [
-            { type = "software-management", path = logs_path }
-        ]
-    };
+    let data = format!(
+        "files = [
+            {{ type = \"software-management\", path = \"{logs_path}\" }},
+        ]"
```

Comment:
In the code base, it looks weird, but it actually prints the better format to end user.
```suggestion
        "files = [
    {{ type = \"software-management\", path = \"{logs_path}\" }},
]"
```

Reply:
fixed, I fixed the `c8y-configuration-plugin.toml` indentation issue as well.